bin/git-stats-importer

```diff
@@ -52,7 +52,7 @@ myRepo.exec("config user.email", function (err, GIT_EMAIL) {
               , all: true
               , "no-color": true
             }, "log");
-            myRepo.exec(command, function (err, data) {
+            myRepo.exec(command + " --pretty=medium --date=default", function (err, data) {
```

Comment:
Yes, it was a bug in oargv. There was no way to configure the separator. :smile: